lamin_cli/_settings.py

```diff
@@ -29,10 +29,10 @@ def settings(ctx):
 )
 @click.argument("value", type=click.BOOL)
 def set(setting: str, value: bool):
-    """Update settings.
+    """Set LaminDB behavior flags.
```

Comment:
Let's please either keep the old wording or choose something better.

"behavior flag" is to me a pretty confusing synonym for a setting 😆